android/capacitor/build.gradle

```diff
@@ -25,4 +25,5 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'com.android.support.test:runner:1.0.1'
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.1'
+    compile 'org.apache.cordova:framework:7.0.0'
```

Comment:
`compile` is deprecated, does `implementation` work as is? That's what it probably should be changed to: https://stackoverflow.com/a/44493379/32140